### PR TITLE
29748 Screen reader texts: translators comment

### PR DESCRIPTION
**Pull request by kebbet**
Core ticket:
https://core.trac.wordpress.org/ticket/29748

### CODE REVIEW

**Comment by SergeyBiryukov** [comment]
Thanks for the PR!

Pretty much all of the comments in core are capitalized and end with a period, so let's do the same here:
```
/* translators: Hidden accessibility text. */
```

There are also these comments previously used for the same purpose (32 matches in 16 files)
```
/* translators: Accessibility text. */
```
Let's change them to use new wording too, for consistency.

**Comment by kebbet** [comment]
> Thanks for the PR!
> 
> 
> 
> Pretty much all of the comments in core are capitalized and end with a period, so let's do the same here:
> 
> ```
> 
> /* translators: Hidden accessibility text. */
> 
> ```
> 
> 
> 
> There are also these comments previously used for the same purpose (32 matches in 16 files)
> 
> ```
> 
> /* translators: Accessibility text. */
> 
> ```
> 
> Let's change them to use new wording too, for consistency.

Thank you for the review. I will adjust the PR later tonight.

**Comment by kebbet** [comment]
@SergeyBiryukov pr updated in response to your comment. Maybe ready for final review and commit?

File: src/wp-admin/update-core.php [hunk 1]
```diff
@@ -748,7 +748,7 @@ function list_theme_updates() {
 				<input type="checkbox" name="checked[]" id="<?php echo $checkbox_id; ?>" value="<?php echo esc_attr( $stylesheet ); ?>" />
 				<label for="<?php echo $checkbox_id; ?>" class="screen-reader-text">
 					<?php
-					/* translators: %s: Theme name. */
+					/* translators: %hidden accessibility text. %s: Theme name. */
```
Comment (costdev) [inline src/wp-admin/update-core.php]:
```suggestion
					/* translators: Hidden accessibility text. %s: Theme name. */
```
Looks like a small mistake here 🙂

**Review by costdev — APPROVED** [review]
Thanks for the update @kebbet! LGTM :+1:

**Comment by SergeyBiryukov** [comment]
Thanks for the PR!

I have found a few misaligned closing tags and tried to correct them on commit, also made some minor formatting edits for consistency. Looks great to me otherwise 🙂 

Merged in [r55276](https://core.trac.wordpress.org/changeset/55276).

**Pull request closed without merging**